docs/storage.md

```diff
@@ -471,6 +472,68 @@ config:
   directory: ""
 ```
 
+### OCI - Oracle Cloud Infrastructure
+
+To configure OCI Object Storage as Thanos Object Store, you need access to OCI Object Storage which supports three different providers, default, instance principal, and raw. 
+The default HTTP Client comes with this Thanos OCI Object Storage should be able to handle most of the use cases.
```

Comment:
Let's try to make this more like a statement. Does it work or not? As a user, you don't want the "should" and "most of the use-cases" :D 

If it does work, we can rename this part towards something like "OCI has 3 different providers; x, y, and z, which are supported by this object storage client implementation" - Or something like that. 

It could also be I'm misreading this part, so feel free to correct me on that.

If there are any docs, I still think it's good to link towards that as well.

Reply:
Make sense Just pushed an updated.